### PR TITLE
Evironment setup

### PR DESCRIPTION
**Pull request by gdumani**
This milestone is about Webpack setup for the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html/CSS
+- Javascript
+- Webpacks
```
Comment (Mihndim2020) [inline README.md:15]: - [ ]  Please your project is missing the following requirements;

req
Kindly implement them and resubmit for review

File: README.md [hunk 2]
```diff
@@ -43,17 +42,11 @@ To get a local copy up and running follow these simple example steps.
```
Comment (Mihndim2020) [inline README.md:51]: - [ ] Please kindly add step to clone and run you project locally.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
 
-# Project Name
+# TODO LIST
 
-> One paragraph statement about the project.
+> "TODO List is simple task organizer written in javascript.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
```
Comment (Mihndim2020) [inline README.md:9]: - [ ] Kindly delete `node_modules` directory from your repository and make sure you add it to `gitignore` file as it not necessary and it makes your repository to be crowded.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Webpack is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import './style.css';
```
Comment (petudeveloper) [inline src/index.js:1]: Even when your style looks good it doesn't look like the one of the video. Please try adding the corresponding changes to look like the one on the video. It seems that your CSS is not being loaded by Webpack Style/CSS Loader.

Check the image for better understanding:
Capture

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+const testList = [
+  { task: 'Do dishes', completed: false, index: 1 },
+  { task: 'Do laundry', completed: false, index: 3 },
+  { task: 'Walk dog', completed: false, index: 2 },
+];
+
+function displayList(l) {
```
Comment (petudeveloper) [inline src/index.js:9]: On page load, the task array is not rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task. Please make the corresponding changes to make your list of tasks be rendered.

Check the image below for a better understanding:
Capture2

**Review by petudeveloper — CHANGES_REQUESTED** [review]
Hi @gdumani, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 5 above)
Comment (gdumani) [reply src/index.js:9]: **If I follow these steps it looks ok to me:**

- git clone https://github.com/gdumani/todolist.git
- cd todolist
- git checkout development
- npm install
- npm run start

I get this image
image

**Review by Cathella — APPROVED** [review]
## 🟩 Status: Approved 🔥

### Hi @gdumani 👋🏻, 

You did a tremendous job implementing the requested changes 👏🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gdumani** [comment]
merge stucture to master

**Pull request merged**